app/scripts/modules/core/src/pipeline/config/stages/wercker/werckerStage.html

```diff
@@ -0,0 +1,188 @@
+<!--
```

Comment:
We don't have copyright notices in any other templates in deck so I'm not sure about including these here. I also notice that you didn't include one in `werckerExecutionDetails.html`.  If these aren't necessary then I suggest removing them.